baker/algolia/configureAlgolia.ts

```diff
@@ -47,13 +47,18 @@ export const configureAlgolia = async () => {
     await chartsIndex.setSettings({
         ...baseSettings,
         searchableAttributes: [
-            "title",
+            "unordered(title)",
+            "unordered(slug)",
```

Comment:
what's your intuition for adding this attribute?

Reply:
It's mostly a "it can't hurt" :)

I realize that in most cases, there is a very high overlap between slug and title.
But one example where this one helps would for example be [Number of deaths from air pollution](https://ourworldindata.org/grapher/total-deaths-from-air-pollution), with slug `total-deaths-from-air-pollution`.
When including the slug, we will get that chart as a response for `air pollution total deaths`.

And I think I also saw some Covid charts where I thought "ah, the slug has some relevant terms in it that the title doesn't have".